src/pages/Login.js

```diff
@@ -12,7 +12,7 @@ function Login() {
                 <input className="input input-primary" type="text" placeholder="Type Username" id="userName"/>
                 <label for="Password">Password*</label>
                 <input className="input input-primary" type="password" placeholder="Type Password" id="password"/>
-                <a href="">Forgot password?</a>  
+                <a href='../'>Forgot password?</a>  
             </div>
             <button className="btn btn-secondary">Login</button>
             <p className="no-account">Don't have an account? <Link to="/signup">Sign Up</Link></p>
```

Comment:
use `htmlFor` instead of `for`
use `Link` instead of `a` tag
Can be done in a future PR

Reply:
Will keep that in mind